UPGRADING.md

```diff
@@ -19,19 +19,23 @@ config.setEndpoints("http://localhost:9339/notify", "http://localhost:9339/sessi
 
 ### Cucumber steps changed
  
-Several Cucumber steps have changed their wording:
+Several Cucumber steps have changed their wording or been split into separate steps:
 
-Old step | New step
+Old step | New step(s)
 ----| -------- | 
-I wait to receive a request | I wait to receive an error
-I wait to receive {int} request(s) | I wait to receive {int} error(s)
-I discard the oldest request | I discard the oldest error
-I should receive no requests | I should receive no errors
-the {string} header is not null | the error {string} header is not null
+I wait to receive a request | I wait to receive an error <br> I wait to receive a session
+I wait to receive {int} request(s) | I wait to receive {int} error(s) <br> I wait to receive {int} session(s)
+I discard the oldest request | I discard the oldest error <br> I discard the oldest session
+the {string} header is not null | the error {string} header is not null <br> the session {string} header is not null
+the {string} header equals {string} | the error {string} header equals {string} <br> the session {string} header equals {string}
+the {string} header matches the regex {string} | the error {string} header matches the regex {string} <br> the session {string} header matches the regex {string}
+the {string} header equals one of: | the error {string} header equals one of: <br> the session {string} header equals one of:
+the {string} header is a timestamp | the error {string} header is a timestamp <br> the session {string} header is a timestamp
 the {string} query parameter equals {string} | the error {string} query parameter equals {string}
 the {string} query parameter is not null | the error {string} query parameter is not null
 the {string} query parameter is a timestamp | the error {string} query parameter is a timestamp
```

Comment:
I don't think so.... I think it could potentially create more work than it saves and introduce more ambiguity in what the steps are doing.  With a half-decent IDE it was really easier for me to do a find/replace over all `.feature` files for Cocoa, which handled 90+% of the migration.  Also, I kind of want it to break obviously so that attention is paid to the detail and an residue changes that are needed are flushed out.

Reply:
Yeah that's fair. I was thinking that we'd be able to add these steps without a major version but that wouldn't work without adding a `Server.requests` BC layer, so it's probably not worthwhile

Reply:
I'm not afraid of major version bumps either, in fact it has the benefit of taking a little more risk on changes, in the knowledge that notifiers will typically only track the latest within a major version stream.